configs/terraform/environments/prod/dns-cleaner.tf

```
@@ -0,0 +1,22 @@
+resource "google_service_account" "sa_gke_kyma_integration" {
+  account_id   = "sa_gke_kyma_integration@sap-kyma-prow.iam.gserviceaccount.com"
+  display_name = "sa_gke_kyma_integration"
+}
+
+resource "google_project_iam_binding" "view_container_analysis_ccurrences" {
```

The resource name sounds to generic. If another identity will need to get same IAM role assigned we will have to use different resource name than `view_container_analysis_ccurrences`. The name of resource should be more specific and provide more context. Maybe something like.

```suggestion
resource "google_project_iam_binding" "dns_cleaner_view_container_analysis_occurrences" {
```

This is the same for bindings of `dns_reader` and `bucket_get` roles.